static/app/components/events/eventCustomPerformanceMetrics.tsx

```diff
@@ -22,13 +22,18 @@ type Props = {
   organization: Organization;
 };
 
+function isNotMarkMeasurement(field: string) {
+  return !field.startsWith('mark.');
```

Comment:
This could also be a comprehensive list of previous mark.* measurements, but few enough people are using custom measurements that I don't anyone is using specifically `mark.` though.